src/app/create-project/create-project.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="new-project-form">
+  <h2> Create a new project</h2>
+  <form>
+    <div class="form-group">
+      <label for="name" class="form-label">Project Name</label>
+      <input #projectName required minlength="3" type="text" class="form-control" ngModel name="name" #name="ngModel"
+             id="name" placeholder="My New Project">
+    </div>
+    <div class="form-group">
+      <label for="api" class="form-label">API base url</label>
+      <input #api type="url" class="form-control" id="api" placeholder="/isAlive">
+    </div>
+    <div class="form-group">
+      <label for="general" class="form-label">General Description</label>
+      <input #desc type="text" class="form-control" id="general"
+             placeholder="Checks if the url sends a ping back">
+    </div>
+    <div class="form-group">
+      <button
+        *ngIf="!(name.invalid && (name.dirty || name.touched) && (name.errors.minlength || name.errors.required)) && projectName.value.length !== 0"
+        (click)="addProject([projectName.value, api.value, desc.value])" type="submit" class="btn btn-primary"><a
+        class="btn btn-primary" href="/viewProjects">
+        Create </a></button>
+    </div>
+    <div *ngIf="name.invalid && (name.dirty || name.touched)"
+         class="alert alert-danger">
+      <div *ngIf="name.errors.minlength">
+        Project name must be at least 3 characters long.
```

Comment:
a. form validation should appear below the respective control, not in the end of the form
b. invalid and touched controls should have indication that something is wrong, like a red border for example...
tip: Invalid controls have classes that indicate it. Use them